### DISCUSSION TITLE
Questions about the pipeline

### DISCUSSION THREAD

**Discussion by xwkya** [voicepaw/so-vits-svc-fork#888] [Q&A]
SO: https://arxiv.org/abs/2111.02392
VITS: https://arxiv.org/abs/2106.06103

Can anyone explain the training and inference pipeline?
What are the components that are actually being trained when running the train command, what are the components that are already pretrained?

Thanks

**Comment by 34j** [comment]
Sorry, but I cannot explain in detail now.
It may be helpful to refer to the Wiki  (where I described the model structure) or the code.
